### DISCUSSION TITLE
How to make this kind of terminal?

### DISCUSSION THREAD

**Discussion by EstopaceMA** [microsoft/terminal#14502] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37571727/206070998-bc7e8277-ba90-45c3-aed5-91ab6595cc2e.png)

**Comment by zadjii-msft** [comment, marked as answer]
Are you talking about the prompt itself? Or the appearance - the purple with the acrylic?

I might recommend checking out [the docs](https://learn.microsoft.com/en-us/windows/terminal/tutorials/custom-prompt-setup)

**Reply by EstopaceMA** [reply]
That's what I'm looking for, Thanks brader!
